lgtm.py

```diff
@@ -280,17 +280,24 @@ def create_from_file() -> 'LGTMSite':
 
 
 @dataclass
+# TODO: this SimpleProject is no longer 'simple'. Some refactoring here could be nice.
 class SimpleProject:
     display_name: str
     key: str
-    is_protoproject: bool
+    project_type: str
+    is_valid_project: bool
+    org: str
+    state: str
 
     def make_post_data(self):
-        data_dict_key = 'protoproject_key' if self.is_protoproject else 'project_key'
+        data_dict_key = 'protoproject_key' if self.is_protoproject() else 'project_key'
         return {
             data_dict_key: self.key
         }
 
+    def is_protoproject(self):
+        # The values for project_type should be hardcoded in one central location
+        return self.project_type == "protoproject"
```

Comment:
Are there other project types other than protoprojects and non-protoprojects?

Reply:
As far as I know, no. But I will take a deeper look.